pkg/components/aws-ebs-csi-driver/component.go

```diff
@@ -17,6 +17,7 @@ package awsebscsidriver
 import (
 	"fmt"
 
+	api "github.com/fluxcd/helm-controller/api/v2beta1"
```

Comment:
Just a doubt. Should we name it `api`? Won't it be little bit vague? How about something like `helmcontrollerapi`

Reply:
changed this as well.